core/src/main/java/org/apache/iceberg/avro/Avro.java

```diff
@@ -235,7 +235,7 @@ public ReadBuilder rename(String fullName, String newName) {
       return this;
     }
 
-    public ReadBuilder nameMapping(NameMapping newNameMapping) {
+    public ReadBuilder withNameMapping(NameMapping newNameMapping) {
```

Comment:
@rdsr, just want to confirm that you're okay with this change? If it is going to cause you problems because you've already deployed it, we can go with the original.